### PR TITLE
Contribution guide: Formatting + draft PR note

### PR DESCRIPTION
**Pull request by DefinitelyMaybe**
Related issues:

#19574

**Description**

Remake of related PR.

### CODE REVIEW

**Review by Mugen87 — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -11,65 +10,74 @@ with git](https://help.github.com/en/github/using-git) and [here’s some help t
...
-* If end-to-end test failed in Travis and you are sure that all is correct, make a new screenshots with npm run make-screenshot <example_1_name> ... <example_N_name> .
+* If you modify files in `examples/js` directory, then don't perform any changes in the `examples/jsm`, JavaScript modules are auto-generated via running `node utils/modularize.js`.
+* If end-to-end test failed in Travis and you are sure that all is correct, make a new screenshots with 
+
+      npm run make-screenshot <example_1_name> ...<example_N_name>
+
 * Watch out for Closure compiler warnings when building the libs, there should not be any.
 * Once done with a patch / feature do not add more commits to a feature branch
 * Create separate branches per patch or feature.
+* If you make a PR but it is not actaully ready to be pulled into the dev branch, add `[Draft]` into the PR title and/or convert it to a draft PR
```
Comment (mrdoob) [inline .github/CONTRIBUTING.md:81]: Nit: `actaully`

Comment (DefinitelyMaybe) [reply .github/CONTRIBUTING.md:81]: good catch. updating.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -11,65 +10,74 @@ with git](https://help.github.com/en/github/using-git) and [here’s some help t
...
 
 As per the npm standard, ‘start’ is the place to begin the package.
 
     npm start
 
-This script will start a local server similar to three.js, but instead will be hosted on your local
-machine. Browse to http://localhost:8080/ to check it out. It also automatically creates the
-‘build/three.js’ and ‘build/three.module.js’ scripts anytime there is a change within your three.js
-directory.
+This script will start a local server similar to three.js, but instead will be hosted on your local machine. Browse to http://localhost:8080/ to check it out. It also automatically creates the ‘build/three.js’ and ‘build/three.module.js’ scripts anytime there is a change within your three.js directory.
```
Comment (mrdoob) [inline .github/CONTRIBUTING.md:29]: > This script will start a local server similar to three.js

What does this mean? Does it mean similar to https://threejs.org?

Comment (DefinitelyMaybe) [reply .github/CONTRIBUTING.md:29]: yup. oh, right, wording. I'll change it to:

local server similar to [threejs.org](https://threejs.org/),

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**